src/app/core/theme.component.ts

```diff
@@ -9,36 +9,110 @@ import { CircularButtonComponent } from '@shared/ui/circular-button/circular-but
-      <app-circular-button icon="close" size="md"></app-circular-button>
-      <app-circular-button icon="close" size="lg"></app-circular-button>
-      <app-circular-button icon="close" size="xl"></app-circular-button>
-      <app-circular-button icon="arrow_back" size="sm"></app-circular-button>
-      <app-circular-button icon="arrow_back" size="md"></app-circular-button>
-      <app-circular-button icon="arrow_back" size="lg"></app-circular-button>
-      <app-circular-button icon="arrow_back" size="xl"></app-circular-button>
-      <app-circular-button icon="arrow_forward" size="sm"></app-circular-button>
-      <app-circular-button icon="arrow_forward" size="md"></app-circular-button>
-      <app-circular-button icon="arrow_forward" size="lg"></app-circular-button>
-      <app-circular-button icon="arrow_forward" size="xl"></app-circular-button>
-      <app-circular-button text="click" size="sm" textSize="sm"></app-circular-button>
-      <app-circular-button text="click" size="md" textSize="md"></app-circular-button>
-      <app-circular-button text="click" size="lg" textSize="lg"></app-circular-button>
-      <app-circular-button text="click" size="xl" textSize="xl"></app-circular-button>
-      <app-circular-button text="click" size="xl" textSize="xxl"></app-circular-button>
+    <div class="circular-buttons-container bg-gradient-neutral-3">
+      <p class="text-body-big">Circular Buttons</p>
+      <div class="circular-buttons-container__btns">
+        <div class="circular-buttons-container__btn">
+          <p>Small size</p>
+          <app-circular-button size="sm" textSize="sm"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Medium size</p>
+          <app-circular-button size="md"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Large size</p>
+          <app-circular-button size="lg"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Extra large</p>
+          <app-circular-button size="xl"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Small done</p>
+          <app-circular-button icon="done" size="sm"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Medium done</p>
+          <app-circular-button icon="done" size="md"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Large done</p>
+          <app-circular-button icon="done" size="lg"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>Extra large done</p>
+          <app-circular-button icon="done" size="xl"></app-circular-button>
+        </div>
+        <div class="circular-buttons-container__btn">
+          <p>small close</p>
```

Comment:
Malutka sprawa, ale jeśli wszystkie nazwy piszesz z wielkiej litery to tu można poprawić żeby było spójnie